programmers/20210503/서울에서-김서방-찾기.js

```diff
@@ -0,0 +1,9 @@
+function solution(seoul) {
+    for(var i = 0; i < seoul.length; i++){
+        if (seoul[i] == "Kim"){
+            answer = i;
+        }
+    }
+    var answer = "김서방은 " + answer + "에 있다";
```

Comment:
자바스크립트에서는 `(백탭)을 이용해서 +를 쓰지 않고도 문장안에 변수를 넣을 수 있습니다. 변수를 넣을때는 $기호를 먼저쓰고 뒤에 {}대괄호를 이용해서 변수를 삽입하시면 됩니다.
```suggestion
    answer = `김서방은 ${answer}에 있다`;
```